content/docs/forwarding-refs.md

```diff
@@ -20,3 +20,8 @@ This means that refs intended for our `FancyButton` component will actually be a
 Fortunately, we can explicitly forward refs to the inner `FancyButton` component using the `React.forwardRef` API. `React.forwardRef` accepts a render function that receives `props` and `ref` parameters and returns a React node. For example:
 `embed:forwarding-refs/log-props-after.js`
 
+## Customizing Name in DevTools
+
+`React.forwardRef` accepts a render function. You can give the render function a better display name in DevTools.
```

Comment:
Maybe we could make this section a little more detailed?

---

`React.forwardRef` accepts a render function. React DevTools uses this function to determine what to display for the ref forwarding component.

For example, the following component will appear as "_ForwardRef_" in the DevTools:
```js
const WrappedComponent = React.forwardRef((props, ref) => {
  return <LogProps {...props} forwardedRef={ref} />;
});
```

If you name the render function, DevTools will include its name (e.g. "_ForwardRef(myFunction)_"):
```js
const WrappedComponent = React.forwardRef(
  function myFunction(props, ref) {
    return <LogProps {...props} forwardedRef={ref} />;
  }
);
```

You can even set the function's `displayName` property to include the component you're wrapping:
```js
function logProps(Component) {
  class LogProps extends React.Component {
    // ...
  }

  function forward(props, ref) {
    return <LogProps {...props} forwardedRef={ref} />;
  }

  // Give this component a more helpful display name in DevTools.
  // e.g. "ForwardRef(logProps(MyComponent))"
  const name = Component.displayName || Component.name;
  forward.displayName = `logProps(${name})`;

  return React.forwardRef(forward);
}
```